### PR TITLE
[F] Implement slack button

### PR DESCRIPTION
**Pull request by MollyJeanB**
Also adjust gradient and add emojis to SplashText

Screenshot 2023-05-26 at 9 04 35 AM
Screenshot 2023-05-26 at 9 04 53 AM

### CODE REVIEW

**Review by bjamba — CHANGES_REQUESTED** [review]
This isn't in scope of the PR but looking at this, I noticed a typo in our copy in the first paragraph. Should be:

> Empathetech (pronounced "empathetic") is an empathy-driven community

We're missing the "an" in the above

**Review by John4850 — APPROVED** [review]
Looks clean

File: src/components/splashText/SplashText.tsx
Comment (John4850) [inline src/components/splashText/SplashText.tsx:1]: Emoji bullets are definitely a good idea

**Review by John4850 — APPROVED** [review]

**Comment by MollyJeanB** [comment]
> This isn't in scope of the PR but looking at this, I noticed a typo in our copy in the first paragraph. Should be:
> 
> > Empathetech (pronounced "empathetic") is an empathy-driven community
> 
> We're missing the "an" in the above

Good call @bjamba. Fixed!

**Pull request merged**